### PR TITLE
Magboots for Sangheili Ranger, Kig-Yar Ranger and Jiralhanae EVA

### PR DESCRIPTION
**Pull request by 5crownik007**
🆑 5crownik007
bugfix: Sangheili Ranger boots now act as magboots. Sangheili Ranger helm now has major flash protection.
rscadd: Kig-Yar Rangers and Jiralhanae EVA now have boots of their own(magboots)
bugfix: Marine EVA boots icon no longer breaks when state is toggled.
/🆑

### CODE REVIEW

File: code/modules/halo/covenant/species/kigyar/kigyar_ranger.dm [hunk 1]
```diff
@@ -22,6 +22,23 @@
...
+	item_state = "ranger_boots"
+	sprite_sheets = list("Kig-Yar" = 'code/modules/halo/covenant/species/kigyar/ranger_kigyar.dmi',"Tvaoan Kig-Yar" = 'code/modules/halo/covenant/species/kigyar/ranger_tvoan.dmi')
+	icon_base = null
+	armor = list(melee = 40, bullet = 35, laser = 35, energy = 35, bomb = 20, bio = 100, rad = 35)
+	body_parts_covered = LEGS|FEET
+	cold_protection = LEGS|FEET
+	heat_protection = LEGS|FEET
+	min_cold_protection_temperature = SPACE_SUIT_MIN_COLD_PROTECTION_TEMPERATURE
+	max_heat_protection_temperature = FIRESUIT_MAX_HEAT_PROTECTION_TEMPERATURE
+	armor_thickness = 15
```
Comment (X0-11) [inline code/modules/halo/covenant/species/kigyar/kigyar_ranger.dm:40]: keep at 20, no need to lower this

File: code/modules/halo/clothing/marine.dm [hunk 2]
```diff
@@ -359,6 +359,7 @@
 	icon_override = MARINE_OVERRIDE
 	item_state = "evalegs"
 	icon_state = "evalegs"
+	icon_base = null
```
Comment (caelaislinn) [inline code/modules/halo/clothing/marine.dm:362]: Did you test this?

File: code/modules/halo/covenant/species/sangheili/clothing.dm [hunk 3]
```diff
@@ -392,16 +392,34 @@
...
+/*/obj/item/clothing/shoes/sangheili/ranger
+	name = "Sanghelli Leg Armour (Ranger)"
+	desc = "Leg armour, to be used with the Sangheili Combat Harness."
+	icon = SANGHEILI_ARMOUR_ICON
+	icon_state = "ranger_legs_obj"
+	item_state = "ranger_legs"
+	item_flags = STOPPRESSUREDAMAGE|THICKMATERIAL|AIRTIGHT
+	body_parts_covered = LEGS|FEET
+	cold_protection = LEGS|FEET
+	min_cold_protection_temperature = SPACE_SUIT_MIN_COLD_PROTECTION_TEMPERATURE*/
```
Comment (caelaislinn) [inline code/modules/halo/covenant/species/sangheili/clothing.dm]: This can be just deleted, the magboot code above makes this all redundant

**Review by caelaislinn — CHANGES_REQUESTED** [review]

File: code/modules/halo/covenant/species/kigyar/kigyar_ranger.dm (hunk 1 above)
Comment (5crownik007) [reply code/modules/halo/covenant/species/kigyar/kigyar_ranger.dm:40]: it's 15 because all the other kig ranger stuff is 15

Comment (X0-11) [reply code/modules/halo/covenant/species/kigyar/kigyar_ranger.dm:40]: *why*

File: code/modules/halo/clothing/marine.dm (hunk 2 above)
Comment (5crownik007) [reply code/modules/halo/clothing/marine.dm:362]: Yes, it works

File: code/modules/halo/covenant/species/kigyar/kigyar_ranger.dm (hunk 1 above)
Comment (5crownik007) [reply code/modules/halo/covenant/species/kigyar/kigyar_ranger.dm:40]: I don't know, I didn't program the other kig ranger equipment, I just followed the example

File: code/modules/halo/covenant/species/sangheili/clothing.dm (hunk 3 above)
Comment (5crownik007) [reply code/modules/halo/covenant/species/sangheili/clothing.dm]: Removed

**Review by caelaislinn — APPROVED** [review]

**Pull request merged**